prime-router/src/test/kotlin/azure/TokenFunctionTests.kt

```diff
@@ -221,8 +221,9 @@ class TokenFunctionTests {
         assertThat(error.get("error_description").textValue()).isEqualTo("malformed_jwt")
         assertThat(error.get("error_uri").textValue()).isEqualTo("$OAUTH_ERROR_BASE_LOCATION#malformed-jwt")
         verify {
+            // per feedback, just fix the assertion based on kotlin compiler NPE handling behavior
```

Comment:
This comment is not very helpful without the context of this ticket. Do we really need this comment? My vote is to just remove it.

Reply:
might be good to log some thinking in the zenhub for future improve..., will remove it